settings/address-levels.json

```diff
@@ -110,16 +110,16 @@
           "county" : [12, 0]
       },
       "boundary" : {
-          "administrative5" : [10, 0]
+          "administrative5" : 10
```

Comment:
Als ich das zuerst getestet hatte, hatte ich den gesamten Block für `de` entfernt. Wenn ich aber die [Dokumentation](https://nominatim.org/release-docs/latest/develop/Ranking/#rank-configuration) richtig verstehe, sollte es reichen, die 0 für "address rank" anzupassen. Das habe ich hier also gemacht.

Ich würde das gerne, zumindest für Deutschland, nochmal genauer testen, nur dauert der Import lokal ca. zwei Tage. Ich könnte es also Freitagabend nochmal starten und dann am Montag testen.

Reply:
hast du das nochmal getestet? in jedem Fall sollte das aber helfen, so wie du schon beschreibst, weiß gerade nicht, wie ich das testen soll ohne einen mordsmäßig großen Import zu machen ^^

Reply:
> hast du das nochmal getestet?

Nein, ich hatte es leider vergessen...